src/memory/primitives/maybeRelocatable.test.ts

```diff
@@ -0,0 +1,72 @@
+import { test, expect, describe } from 'bun:test';
+import { Felt } from './felt';
+import {
+  ForbiddenOperation,
+  OffsetUnderflow,
+  Relocatable,
+} from './relocatable';
+import { add, sub } from './maybeRelocatable';
+import { InternalError } from '../types';
+
+describe('maybe relocatable', () => {
```

Comment:
describe should describe the System Under Testint (SUT), here the type `MaybeRelocatable`, hende 

```
describe("MaybeRelocatable")
```